ompi/mca/coll/hcoll/coll_hcoll_component.c

```diff
@@ -251,6 +251,8 @@ static int hcoll_open(void)
 
     cm->libhcoll_initialized = false;
 
+    (void)mca_base_framework_open(&opal_memory_base_framework, 0);
```

Comment:
@jsquyres just like in opal/rcache - the next line of code calls  opal_mem_hooks_support_level() to detect if memory hooks are supported, so no need to check the return value from framework open call.